cmd/schema.go

```diff
@@ -306,24 +289,25 @@ func getSchemaCmd() *cobra.Command {
 
 			fmt.Printf("Schema successfully written to %s\n", output)
 
-			if resMetadata != nil {
-				report := [][]string{}
+			if metadata {
```

Comment:
@ishanarya0 These two if can be combined into one, right?

Reply:
Yes, done.